src/op_mode/firewall.py

```diff
@@ -89,10 +90,38 @@ def get_nftables_details(family, hook, priority):
         out[rule_id] = rule
     return out
 
-def output_firewall_vertical(rules, headers):
+def get_nftables_group_members(family, table, name):
+    prefix = 'ip6' if family == 'ipv6' else 'ip'
+    out = []
+
+    try:
+        results_str = cmd(f'sudo nft -j list set {prefix} {table} {name}')
```

Comment:
Should we have a function for easily calling NFTables with JSON output?

Reply:
Good idea for a vyos.utils helper